sdk/core/azure-core-serializer-json-jackson/README.md

```diff
@@ -1,6 +1,6 @@
-# Azure Core Jackson JSON Serializer plugin library for Java
+# Azure Search Jackson JSON Serializer plugin library for Java
 
-Azure Core Jackson JSON Serializer is a plugin for the `azure-core` `JsonSerializer` API.
+Azure Search Jackson JSON Serializer is a plugin for the `azure-search-documents` `JsonSerializer` API.
```

Comment:
Revert these as this is the Core Jackson JSON Serializer.

Reply:
Good catch